v2/pkg/genruntime/configmaps/configmaps.go

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package configmaps
```

Comment:
Why have a separate nested package like this, especially one that essentially contains just a single helper function?

Reply:
It's going to get more stuff in future PRs. For example `configmaps.Collector` (similar to `secrets.Collector` we already have).

Since `genruntime` is already pretty big, I'm trying to (where it makes sense) make subpackages. Some stuff still has to go at the top level so it's not like genruntime is going to stop growing, but trying to combat bloat some by putting things like `configmaps.Collector`, `configmaps.SliceToClientObjectSlice`, `secrets.Collector`, etc into their own sub packages. It also feels more idiomatic in Go than having `genruntime.ConfigMapCollector` and `genruntime.SecretCollector`.

Reply:
Fair call. :+1: